applications/FluidDynamicsApplication/custom_utilities/embedded_discontinuous_data.h

```diff
@@ -0,0 +1,109 @@
+//    |  /           |
```

Comment:
format this file. you are adding it new, and it will be easier to read

Reply:
Ok!

Reply:
Done.